monitoring/mixin.libsonnet

```diff
@@ -0,0 +1,15 @@
+(import 'config.libsonnet') +
+(import 'alerts/vgalerts.libsonnet') + {
+  prometheus+:: {
+        apiVersion: 'monitoring.coreos.com/v1',
+        kind: 'PrometheusRule',
+        metadata: {
+          name: 'prometheus-vg-rules',
+          namespace: 'default',
```

Comment:
namespace should not be default here. I think kustomize looks for system.

Reply:
Done